### PR TITLE
Additional functions to win_iis module

### PR DESCRIPTION
**Pull request by twangboy**
### What does this PR do?
Adds the following functions to the win_iis execution module:
- modify_site
- modify_binding
- list_backups
- create_backup
- remove_backup
- list_worker_processes

### What issues does this PR fix or reference?
https://github.com/saltstack/zh/issues/1049

### Tests written?
No

### CODE REVIEW

File: salt/modules/win_iis.py [hunk 1]
```diff
@@ -211,8 +211,8 @@ def create_site(name, sourcepath, apppool='', hostheader='',
         name (str): The IIS site name.
         sourcepath (str): The physical path of the IIS site.
         apppool (str): The name of the IIS application pool.
-        hostheader (str): The host header of the binding. Usually the hostname
-            or website name, ie: www.contoso.com
+        hostheader (str): The host header of the binding. Usually the hostname,
+            ie: contoso.com
```
Comment (cachedout) [inline salt/modules/win_iis.py]: This seemed more clear to me in its original form, tbh. This makes it less clear that you're referring to the FQDN versus just the domain name.

File: salt/modules/win_iis.py [hunk 2]
```diff
@@ -277,6 +276,75 @@ def create_site(name, sourcepath, apppool='', hostheader='',
...
+        If an application pool is specified, and that application pool does not
+        already exist, it will be created.
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' win_iis.modify_site name='My Test Site' sourcepath='c:\\new_path' apppool='NewTestPool'
+    '''
+    site_path = r'IIS:\Sites\{0}'.format(name)
```
Comment (cachedout) [inline salt/modules/win_iis.py]: I believe this will resolve to `IIS:\Sites\\my_str`. Is that desired?

File: salt/modules/win_iis.py [hunk 3]
```diff
@@ -1422,3 +1580,160 @@ def remove_vdir(name, site, app=_DEFAULT_APP):
...
+        dict: A dictionary of worker processes with their process IDs
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' win_iis.list_worker_processes 'My App Pool'
+    '''
+    ps_cmd = ['Get-ChildItem',
+              r"'IIS:\AppPools\{0}\WorkerProcesses'".format(apppool)]
```
Comment (cachedout) [inline salt/modules/win_iis.py]: Same comment as above. I think you may end up with an extra forward-slash here.

File: salt/modules/win_iis.py (hunk 2 above)
Comment (twangboy) [inline salt/modules/win_iis.py]: it resolves to `IIS:\\Sites\\my_str`

File: salt/modules/win_iis.py (hunk 3 above)
Comment (twangboy) [inline salt/modules/win_iis.py]: IIS: is like C:, not like http:

**Pull request merged**